ntbs-service/Pages/Shared/_AlertsAndActions.cshtml

```diff
@@ -10,31 +10,40 @@
     @if ((string)ViewData["CurrentPage"] == NotificationSubPaths.Overview)
     {
         <nhs-grid-column grid-column-width="TwoThirds">
-            <div class="@manageNotificationClass">
-                <h2>
-                    Alerts
-                </h2>
-                @if(Model.Alerts.Count > 0)
-                {
-                    <div class="overview-alerts-container">
-                    @foreach (var alert in Model.Alerts)
+            @if(!Model.HasEditPermission)
```

Comment:
It felt cleaner (definitely at the expense of the if loop logic) to have the shorter slightly more confusing case at the top rather than hidden at the bottom but happy to change!

Reply:
Yeah, there's definitely merit to that. I was thinking more around the happy-path and using-positives arguments, but i guess they pretty much balance each other out - happy to keep as is!